### PR TITLE
Change pkg/apis and pkg/client tag in go.mod

### PR DESCRIPTION
**Pull request by lxiaopei**
In go.mod, change pkg/apis and pkg/client tag to v0.0.1.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -22,8 +22,8 @@ require (
 	github.com/prometheus/client_golang v1.16.0
 	github.com/sirupsen/logrus v1.9.0
 	github.com/stretchr/testify v1.8.2
-	github.com/vmware-tanzu/nsx-operator/pkg/apis v1.0.0
```
Comment (jwsui) [inline go.mod:25]: I think the replace in line #5 and #6 could be removed.

Comment (wyike) [reply go.mod:25]: +1

Comment (lxiaopei) [reply go.mod:25]: for nsx-operator build, do you want to use the local file from pkg/apis and pkg/client folder or a specific version?

Comment (jwsui) [reply go.mod:25]: v0.0.1 would be good enough.

Comment (lxiaopei) [reply go.mod:25]: change to use the local files for pkg/apis and pkg/client instead of tags, to avoid changes when tags change.

**Pull request merged**